lib/optimizely/notification_center.rb

```diff
@@ -0,0 +1,150 @@
+#
+#    Copyright 2017, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+module Optimizely
+  class NotificationCenter
+    attr_reader :notifications
+    attr_reader :notification_id
+
+    NOTIFICATION_TYPES = {
+      ACTIVATE: 'ACTIVATE: experiment, user_id, attributes, variation, event',
+      TRACK: 'TRACK: event_key, user_id, attributes, event_tags, event',
+      FEATURE_ROLLOUT: 'FEATURE_ROLLOUT:feature_key, user_id, attributes, audiences',
```

Comment:
As mentioned in our meeting today, please remove `FEATURE_ROLLOUT` and `FEATURE_EXPERIMENT` notification types for now.

Reply:
Removed.